public/doc/swagger-2-v0.0.2.yaml

```diff
@@ -79,6 +79,33 @@ securityDefinitions:
   UserSecurity:
     type: "basic"
 definitions:
+  Container:
+    allOf:
+    - $ref: "#/definitions/InventoryObject"
+    - type: "object"
+      required:
+      - "container_group"
+      - "name"
+      properties:
+        container_group:
+          $ref: "#/definitions/InventoryObjectLazy"
+        name:
+          type: "string"
+        cpu_limit:
+          type: "number"
+          format: "double"
+        cpu_request:
+          type: "number"
+          format: "double"
+        memory_limit:
+          type: "number"
+          format: "double"
```

Comment:
This is a bigint in the schema

Reply:
:+1: you're right, copy&paste error here